driver.py

```diff
@@ -407,14 +445,40 @@ def generate_probe(p: Panel, options: Options) -> str:
 
 	s += '''
 	drm_panel_add(&ctx->panel);
+'''
+
+	if p.has_dsc:
+		s += f'''
+	/* This panel only supports DSC; unconditionally enable it */
+	dsi->dsc = &ctx->dsc;
+
+	ctx->dsc.dsc_version_major = {p.dsc_version};
+	ctx->dsc.dsc_version_minor = {p.dsc_scr_version};
+
+	// TODO: Pass slice_per_pkt = {p.dsc_slice_per_pkt}
```

Comment:
I think upstream might prefer `/* TODO */` (assuming this is meant to be kept for upstreaming at the moment)

Reply:
Good catch.